packages/eas-cli/src/commandUtils/flags.ts

```diff
@@ -0,0 +1,19 @@
+import { Flags } from '@oclif/core';
+
+export const EasNonInteractiveFlags = {
+  nonInteractive: {
+    json: Flags.boolean({
+      description: 'Enable JSON output, non-JSON messages will be printed to stderr.',
+      dependsOn: ['non-interactive'],
```

Comment:
dependsOn requires that other flag is set, but it does not require to be set to a specific value, Are you sure that it's not possible to set `--non-interactive=false` or sth like that?

Reply:
if it's not possible to set it to false, then I think dependsOn is fine